pom.xml

```diff
@@ -367,6 +367,9 @@
           <systemPropertyVariables>
             <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
           </systemPropertyVariables>
+          <additionalClasspathElements>
+            <additionalClasspathElement>${jdbcDriverJars}</additionalClasspathElement>
```

Comment:
If you run a test that doesn't require this and you don't specify the property (ex: the generic db test), does it complain about this property not being defined? If so, it would be better to default this to some value so that it's not always required.

Reply:
done, I provided empty placeholder for that case